_summaries/sukhbaatar2015end-to-end_2.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{sukhbaatar2015end-to-end}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: mmcinnestaylor # Your GitHub id
+score: 8 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+* End-To-End Memory Networks are a type of Memory Network with a possibly large external memory which is are trained end-to-end.
+* End-to-end training requires less supervision, thus improving applicability in real world settings.
+
+## Technical Implementation
+![Architecture](sukhbaatar2015end-to-end_2_b.png) 
+* The model accepts a discrete set of inputs (a story) along with a query, and outputs an answer.  
+![Q&A](sukhbaatar2015end-to-end_2_a.png)  
+* The inputs {x} and query q are transformed using embedding matrices A and B, respectively.  
+$$m_i = \text{Embed}_A(x_i)\qquad u = \text{Embed}_B(q)$$
+* The the most relevant embedded story input is matched with the embedded query by taking the inner product of each input with the query followed by a softmax operation, thus producing a probability vector.  
+$$p_i = \text{softmax}(u^{\text{T}}m_i)$$  
+* The unembedded story inputs are then embedded once again using embedding matrix C.  
+$$c_i = \text{Embed}_C(x_i)$$  
+* The response vector is then calculated by summing the inputs in {c} and weighting each with the probability vector from the previous step.  
```

Comment:
Small formatting issue (possibly): again not sure if you mean {} around c, if you did you can totally ignore this. 
<img width="1056" alt="Screen Shot 2021-09-25 at 3 07 25 PM" src="https://user-images.githubusercontent.com/13022900/134784785-d6e4f131-6d1e-4018-8778-545b3ffbc0cc.png">

Reply:
@Zayne-sprague Same latex issue I mentioned above.

Reply:
Yeah this is really rough because I think its a difference in engines.  I do see nice correct formatting with $$xyz$$, but it's always hard to publish something if it doesn't look right on your end.  

If you want to change all the {} to $$ $$ and then send me a message, I can render it using Jekyll and post a pdf version so you can see it?

Reply:
@Zayne-sprague I committed a change doing just that.